### ISSUE TITLE
3-4 macOS functional test failures on master - "Unable to register repo because GVFS.Service is not responding."

### ISSUE THREAD

**Issue by pmj** [microsoft/VFSForGit#1404]
I was getting 3 identical, new functional test failures on different PRs, which didn't seem right so I kicked off a test run against master, and that now also fails.

The failing tests are:

* `GVFS.FunctionalTests.Tests.EnlistmentPerFixture.MountTests.MountCanProcessSavedBackgroundQueueTasks()`
* `GVFS.FunctionalTests.Tests.EnlistmentPerFixture.DehydrateTests.DehydrateShouldFailIfGitObjectsRootNotInMetadata()`
* `GVFS.FunctionalTests.Tests.EnlistmentPerFixture.CloneTests.CloneInsideUnmountedEnlistment()`

They all fail because the command output contains "`WARNING: Unable to register repo because GVFS.Service is not responding.`"
(The failing tests are different tests than #1204, hence tracking this as a new issue. Repro rate also seems much higher. Could be related of course, I can't tell.)

Retained logs:
 * https://dev.azure.com/mseng/AzureDevOps/_build/results?buildId=10084307&view=results (PR #1208)
 * https://dev.azure.com/mseng/AzureDevOps/_build/results?buildId=10083825&view=results (PR #1280)
 * https://dev.azure.com/mseng/AzureDevOps/_build/results?buildId=10089649&view=results (master @ 34ed486) - on this run, 4 tests failed, the same 3 as above plus `GVFS.FunctionalTests.Tests.EnlistmentPerFixture.MountTests.MountChangesMountId()`, again for the same reason.

I don't really understand these test failures so I'd appreciate if someone more knowledgeable could take a look. Thanks!

**Comment by pmj** [comment]
Note that repro rate is not 100%, I rebased PR #1280 on latest master earlier, and that run completed successfully: https://dev.azure.com/mseng/AzureDevOps/_build/results?buildId=10089731&view=results

**Comment by derrickstolee** [comment]
We had these happening early last week, and @jamill root-caused it to a long list of repos registered for auto-mount. I believe he is working on removing those repos before starting the functional test suite.

I see that all of your failures are on the GVFS-MACPRO-2 machine. Maybe that machine is overloaded already?

**Comment by jamill** [comment]
I cleaned up the bad state on GVFS-MACPRO-2, we will keep an eye on it to make sure it is healthy again. I am looking at fixing the underlying issue.

**Comment by pmj** [comment]
Thank you!

**Comment by jamill** [comment]
Resolved with #1417
